src/docs/scripts/reference.py

```diff
@@ -1,47 +1,715 @@
-# Adapted from https://github.com/mkdocstrings/python/blob/main/scripts/gen_ref_nav.py
```

Comment:
The old file was practically removed, it has no connection to the new implementation, only in file name, so it was also ok to remove this license comment.